math/minuit/src/TMinuit.cxx

```diff
@@ -11,6 +11,7 @@
 
 
 /*! \class TMinuit
+\legacy{TMinuit, Consider switching to Minuit2.}
```

Comment:
```suggestion
```
Same comment as for `minuit/doc/index.md`. I think we should not make the old Minuit legacy.